### ISSUE TITLE
Error in logs about BrokenLinkEmailsMiddleware

### ISSUE THREAD

**Issue by jonespm** [tl-its-umich-edu/remote-office-hours-queue#560]
I'm not sure what's generating this INFO message but it should be removed as it provides no useful information.

`INFO 2024-10-23 13:04:24,811 Super class BrokenLinkEmailsMiddleware handling 404 broken link: {request.path}`

**Comment by lsloan** [comment]
So, this was moved 01.01 → 02.01 in order to make the 01.01 release lighter?

**Comment by jonespm** [comment]
Yes, it was moved because 01.01 was getting full but if this is a minor issue and easy to resolve it can be included.
